pkg/manila/const.go

```diff
@@ -79,8 +79,18 @@ const (
 	ShortDuration = time.Duration(5) * time.Second
 	// NormalDuration -
 	NormalDuration = time.Duration(10) * time.Second
-	//DBSyncCommand -
+	// DBSyncJobName -
+	DBSyncJobName = "db-sync"
+	// DBSyncCommand -
 	DBSyncCommand = "/usr/bin/manila-manage --config-dir /etc/manila/manila.conf.d db sync"
+	// SvcCleanupJobName -
+	SvcCleanupJobName = "service-cleanup"
+	// SvcCleanupCommand -
+	SvcCleanupCommand = "/usr/bin/manila-manage --config-dir /etc/manila/manila.conf.d service cleanup"
```

Comment:
Looks like we might want to introduce a delay in the command to make sure the scheduler is able to catch the event (as it reports every 60 seconds).

Reply:
Done